release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2890,6 +2890,47 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.34
+[id="ocp-4-17-34_{context}"]
+=== RHBA-2025:9289 - {product-title} {product-version}.34 bug fix update
+
+Issued: 25 June 2025
+
+{product-title} release {product-version}.34 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2025:9289[RHBA-2025:9289] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:9290[RHBA-2025:9290] advisory.
```

Comment:
this is probably expected behavior but i'm noting it just in case, the advisory links give 404 when trying to access them

Reply:
Hi, Jesse. Yes, this is expected behavior until the advisories are published and then the links are accessible to our customers.